src/app/core/constants/routes.constants.ts

```diff
@@ -24,4 +24,5 @@ export enum CoreRoutes {
     TheWire = 'thewire',
     Unauthorized = 'unauthorized',
     Forbidden = 'forbidden',
+    GCtools ='gctools',

```

Comment:
We don't need a route for GCTools. I think Chaza just wanted to link to our GitHub org [here](https://github.com/gctools-outilsgc). This can be removed as well as the component files.

Reply:
I think the position of the list needs to shift left. It looks fine on a smaller width screen like this

![image](https://user-images.githubusercontent.com/2327968/236879622-0ceac4bd-0e0c-44d3-85ff-a20c58e16bf7.png)

But on a widescreen it's not in-line with the header:

![image](https://user-images.githubusercontent.com/2327968/236879865-1b711995-720a-4747-87b6-c1f718bd5914.png)

I think we should also have the Canada logo anchored to the right.

Reply:
Also try adding this to the CSS for the image `filter: brightness(0) invert(1);`

It should make the logo all white.